webhook_to_fedora_messaging/endpoints/apikey.py

```diff
@@ -0,0 +1,86 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.apikey import APIKey
+from ..models.user import User
+from datetime import datetime
+from sqlalchemy_helpers import get_or_create
+from .util import not_found, success, bad_request, created, conflict, validate_request, exclude_from_val
+
+app = Flask(__name__)
+apikey_endpoint = Blueprint("apikey_endpoint", __name__)
+
+@apikey_endpoint.route("/apikey", methods=["POST"])
+def create_apikey():
+    """
+        Used for creating a new service by sending a post request to /service/ path.
+    """
+    
+    session = db.Session()
+    body = request.json
+    
+    
+    user = session.query(User).filter(User.username == body['username']).first()
+    if user is None:
+        return not_found()
+    # Can be different parsing here.
+    valid_till = datetime.strptime(body['valid_till'], "%Y-%m-%d")
+    apikey, is_created = get_or_create(session, APIKey, user_id=user.id, name=body['name'], expiry_date=valid_till)
+    
+    if not is_created:
+        return conflict({'message': 'Key Already Exists'})
+    else:
+        return created({'message': 'Created', 'uuid': apikey.id, 'code': apikey.token})
+        
+    
+@apikey_endpoint.route("/apikey/search", methods=["GET"])
+def list_apikey():
+    
```

Comment:
Use this empty line ;-)

If docstrings are created for a function of a kind, please ensure that they are created for other functions of that kind.